core/__tests__/models/property/localCache.ts

```diff
@@ -103,20 +106,48 @@ describe("models/property", () => {
   });
 
   describe("#findOneWithCache", () => {
-    let emailProperty: Property;
+    let firstNameProperty: Property;
 
     beforeAll(async () => {
-      emailProperty = await Property.findOne({ where: { key: "email" } });
+      firstNameProperty = await Property.findOne({
+        where: { id: "firstName" },
+      });
+      await firstNameProperty.update({ key: "FIRST NAME" });
+      expect(CachedProperties.expires).toEqual(0);
+    });
+
+    test("after a Property is updated, the local cache should be invalid", async () => {
+      const lastNameProperty = await Property.findOne({
+        where: { key: "lastName" },
+      });
+      await lastNameProperty.update({ key: "LAST NAME" });
+      expect(CachedProperties.expires).toEqual(0);
```

Comment:
Should we set or check that `CachedProperties.expires` is something else before this? I believe it would be `0` anyway at this point.